### ISSUE TITLE
Cant update after I log in on the client. 

### ISSUE THREAD

**Issue by Haladir** [dalaranwow/dalaran-wow#5342]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Every time I log into the game and should come to the character creation place, I get this update which is stuck on 0% and then tells me to relog the game. Than a error msg poops up saying " Failled to apply patch. Please try again later! ... "
**Expected behaviour**: Tell us what should happen instead
Should take me straight to my char creater.
**Steps to reproduce the problem**:
![update](https://user-images.githubusercontent.com/37251755/37246612-e0b72758-24ac-11e8-92ce-f22291635be8.png)
![failed](https://user-images.githubusercontent.com/37251755/37246613-e783db9e-24ac-11e8-82cf-4426d7fa8ca4.png)
![patch needed](https://user-images.githubusercontent.com/37251755/37246614-e7a7d8e6-24ac-11e8-8e29-703893f36325.png)

1. Waited some time.
2. Restarted the pc.
3. Tried to run with Admin and have the same problem.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Make sure that you log in with your account username, and not with your e-mail, and it will work.

Cheers, closing up.
